scripts/setup_colab.sh

```diff
@@ -228,7 +219,7 @@ cp "${_arg_mjkey}" "${HOME}/.mujoco/mjkey.txt"
   pip install -e .[all]
   pip install -e .[dev]
   pip install git+https://github.com/rlworkgroup/metaworlds.git@master#egg=metaworlds
```

Comment:
please ensure that you also upgrade the packages at rlworkgroup/metaworlds and  that the tests in metaworlds still pass. they should be runnable inside the garage docker/conda environments.